src/com_tjvendors/site/events/vendor.php

```diff
@@ -59,9 +60,8 @@ public function onAfterVendorSave($vendorDetails, $isNew)
 				break;
 		}
 
-		$dispatcher = JDispatcher::getInstance();
 		PluginHelper::importPlugin('tjvendors');
-		$dispatcher->trigger('tjVendorsOnAfterVendorSave', array($vendorDetails, $isNew));
+		Factory::getApplication()->triggerEvent('tjVendorsOnAfterVendorSave', array($vendorDetails, $isNew));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done